js/_website/src/routes/[[version]]/docs/themes/+page.server.ts

```diff
@@ -27,7 +27,7 @@ export async function load({ parent }) {
 
 	let objs = [docs.building.base];
 	let headers = [
-		["Base", "base"],
+		["themes.Base", "base"],
```

Comment:
Instead of themes.Base or Base we can just use `Theme`
as you can do gr.Theme.push_to_hub and it looks correct as well.